demos/projects/ESPRESSIF/esp32/components/sample-azure-iot/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+set(ROOT_PATH
+    ${CMAKE_CURRENT_LIST_DIR}/../../../../../..
+)
+
+if (DEFINED CONFIG_AZURE_SAMPLE_USE_PLUG_AND_PLAY)
+    file(GLOB_RECURSE COMPONENT_SOURCES
+        ${ROOT_PATH}/demos/sample_azure_iot_pnp/*.c  
+    )
+else()
+    file(GLOB_RECURSE COMPONENT_SOURCES
+        ${ROOT_PATH}/demos/sample_azure_iot/*.c  
+    )
+endif()
+
+idf_component_get_property(LWIP_DIR lwip COMPONENT_DIR)
```

Comment:
just wondering where you need lwip inc? only place I can think of is transport_lts_esp32.c, but that has no reference to lwip. Am I missing anything?

Reply:
No, indeed that was a left over reference. Removed.